NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Created serializers for three models

## Added Serializers and views for four tables:
1. User
1. Product
1. Employee Training
1. Updated the README

## Steps to Test

1. Run virtual environment
1. Start django app
1. Navigate to http://127.0.0.1:8000/bangazonapi/[tablename]
There are three tables to test
Verify that you can see the data from the database
```
git fetch --all
git checkout JS_creating_other_rest_items
```

## Impacted Areas in Application

##### Modified
bangazonapi/serializers/User_Serializer.py
bangazonapi/serializers/__init__.py
bangazonapi/urls.py

##### Deleted
views/User_View.py
views/_ _init_ _.py

##### to be Tracked
bangazonapi/serializers/Employee_Training_Serializer.py
bangazonapi/serializers/Product_Serializer.py
bangazonapi/views

## Mentions @username
@MrErin @obscure-shadow @pwmurphy89 

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
